helm/templates/schema-compatibility/cronjob.yaml

```diff
@@ -28,17 +28,17 @@ spec:
           volumes:
             - name: service-config
               configMap:
-                name: {{ .Values.schemaCompatibility.config.name }}
+                name: {{ .Values.schemaCompatibility.name }}
           containers:
             - command: [ "/bin/sh", "-c" ]
               args:
                 - |
                   echo "Running python script: compatibility changer"
                   cd /opt/schema-compatibility
                   /opt/venv/bin/python -m cron-job.py
-              image: "{{ .Values.image.registry }}/{{ .Values.schemaCompatibility.image.repository }}:{{ default $.Chart.AppVersion .Values.schemaCompatibility.image.tagOverride }}"
+              image: "{{ .Values.schemaCompatibility.image.repository }}:{{ default $.Chart.AppVersion .Values.schemaCompatibility.image.tagOverride }}"
```

Comment:
why this change, we need registry to handle different regions

Reply:
Hypertrace repos publish to dockerhub, so `hypertrace/schema-compatibility:0.3.2` is the correct image and not `us-docker.pkg.dev/traceable-actions-386017/docker/hypertrace/schema-compatibility:0.3.2`

cc: @ravisingal

Reply:
Ah okay, fine then